### PR TITLE
Prefix `gen_shapshot` build invocation with `time` based on the current platform.

### PR DESCRIPTION
**Pull request by matanlurey**
Work on debugging https://github.com/flutter/flutter/issues/154437.

### CODE REVIEW

File: lib/snapshot/BUILD.gn [hunk 1]
```diff
@@ -63,6 +63,9 @@ group("generate_snapshot_bins") {
 # See: `bin_to_linkable` rules below that build these outputs into linkable form
 # See: https://github.com/flutter/flutter/wiki/Flutter-engine-operation-in-AOT-Mode
 compiled_action("generate_snapshot_bin") {
+  # TODO(https://github.com/flutter/flutter/issues/154437).
+  prefix_with_time_cmd = true
```
Comment (aam) [inline lib/snapshot/BUILD.gn:67]: Does this flag need to be added to https://github.com/flutter/buildroot repo first?

Comment (matanlurey) [reply lib/snapshot/BUILD.gn:67]: Oh shoot, I thought I had sent the buildroot patch, let me send you that first!

Comment (matanlurey) [reply lib/snapshot/BUILD.gn:67]: https://github.com/flutter/buildroot/pull/902

**Review by aam — APPROVED** [review]

File: lib/snapshot/BUILD.gn (hunk 1 above)
Comment (matanlurey) [reply lib/snapshot/BUILD.gn:67]: Rolled buildroot in this PR as well.

**Comment by matanlurey** [comment]
Looks like it assumed the executable was local, versus on the PATH:

```
FileNotFoundError: [Errno 2] No such file or directory: './time'
```

I'll... look at this again.

**Pull request closed without merging**